courses/templates/courses/introduction-to-data-mining.html

```diff
@@ -0,0 +1,148 @@
+                    </p>
+                    <footer>Savannah Gosnell</footer>
+                </blockquote>
+            </div>
+        </div>
+    </div>
+</div>
+
+<h2>Lecture outline and schedule</h2>
+<div class="schedule">
+    <h5>September 11, 2017. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Data exploration</b>. Data preparation for machine learning, introduction to
+        data analysis workflows,
+        visual programming as a means of design of data analysis workflows,
+        basic data exploration and visualization, rigor and reproducibility in data mining.</p>
+
+    <h5>September 12, 2017. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Introduction to classification</b>. Classification trees. Confusion matrix.
+        Estimation of prediction error.
+        Overfitting. Cross-validation.</p>
+
+    <h5>September 13, 2017. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Classification models</b>. Classification trees, forests, logistic regression,
+        nearest neighbors classification.
+        Data painting. Area under ROC curve. Analysis of results of classifications. Nomograms.
+    </p>
+
+    <h5>September 15, 2017. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Regression</b>. Linear regression, polynomial regression and overfitting.
+        Regularization.
+        Effects of regularization on training and test sets.
+        Random forests for regression.</p>
+
+    <h5>September 19, 2015. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Partitional clustering</b>. Introduction to k-means clustering.
+        Differences with hierarchical clustering. Silhouette scoring of clustering results. Outlier detection.</p>
+
+    <h5>September 20, 2017. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Data projections</b>. Principal component analysis. Multi-dimensional scaling
+        and t-SNE.</p>
+
+    <h5>September 22, 2017. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Gentle introduction to deep learning</b>. Image embedding. Image data analysis
+        – examples with clustering,
+        classification and outlier detection with images. Course wrap-up.</p>
+</div>
+<br>
+
+<div class="panel panel-warning">
+    <div class="panel-heading">
+        Lecturers
+    </div>
+
+    <div class="panel-body">
+        <div class="wrapper">
+            <div class="one" style="padding-top:1em">
+                <img style="float: left;"  src="{% static 'courses/janez_demsar_bw3.png' %}" alt="" />
```

Comment:
I would also suggest to move `style="float: left;"` to css file, since it repeats in page.